src/org/mozartoz/truffle/nodes/builtins/BrowserBuiltins.java

```diff
@@ -7,18 +7,19 @@
 import com.oracle.truffle.api.dsl.NodeChild;
 import com.oracle.truffle.api.dsl.NodeChildren;
 import com.oracle.truffle.api.dsl.Specialization;
+import com.oracle.truffle.api.object.DynamicObject;
 
 public abstract class BrowserBuiltins {
 
+	@Builtin(tryDeref = 0)
 	@GenerateNodeFactory
 	@NodeChild("value")
 	public static abstract class IsRecordCVarNode extends OzNode {
 
 		@Specialization
 		Object isRecordCVar(Object value) {
-			return unimplemented();
+			return value instanceof DynamicObject;
```

Comment:
This should also accept lists and atoms. The easiest way to do that is to reuse `IsRecordNode`.